.env.sample

```diff
@@ -2,6 +2,7 @@ BUGZILLA_API_KEY=
 BUGZILLA_CC_LIST=
 BUGZILLA_HOST=https://bugzilla.allizom.org/
 CELERY_SCHEDULE_INTERVAL=300
+COMPOSE_HTTP_TIMEOUT=120
```

Comment:
@jodyheavener Yeah you were right we need this, integration on circle has been complaining about it XD

Reply:
Aha!